sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Microsoft.Azure.Management.DataFactory.csproj

```diff
@@ -5,15 +5,14 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.DataFactory</PackageId>
     <Description>Azure Data Factory V2 is the data integration platform that goes beyond Azure Data Factory V1's orchestration and batch-processing of time-series data, with a general purpose app model supporting modern data warehousing patterns and scenarios, lift-and-shift SSIS, and data-driven SaaS applications. Compose and manage reliable and secure data integration workflows at scale. Use native ADF data connectors and Integration Runtimes to move and transform cloud and on-premises data that can be unstructured, semi-structured, and structured with Hadoop, Azure Data Lake, Spark, SQL Server, Cosmos DB and many other data platforms.</Description>
-    <Version>4.3.0</Version>
+    <Version>4.3.1</Version>
```

Comment:
I suggest you follow sementic versioning -- update the minor version if you are adding new features

Reply:
i.e. 4.4.0

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/7452
https://github.com/Azure/azure-rest-api-specs/pull/7437